security/index.md

```diff
@@ -6,9 +6,41 @@ toc_min: 1
 toc_max: 2
 ---
 
+## CVE-2022-42889
+
+[CVE-2022-42889](https://nvd.nist.gov/vuln/detail/CVE-2022-42889){:target="_blank" rel="noopener" class="_"} was discovered in the popular Apache Commons Text library. Versions of this library up to but not including 1.10.0 are affected by this vulnerability.
+
+We strongly encourage you to update to the [latest version](https://commons.apache.org/proper/commons-text/download_text.cgi) if you can.
+
+### Docker Official Images impacted by CVE-2022-42889
+
+> **Important**
+>
+> We will be updating this section with the latest information. We recommend
+> that you revisit this section to view the list of affected images and update
+> images to the patched version as soon as possible to remediate the issue.
+{: .important}
+
+A number of [Docker Official Images](../docker-hub/official_images.md) contain the vulnerable versions of
+Apache Commons Text. The following table lists Docker Official Images that
+may contain the vulnerable versions of Apache Commons Text. We are working on updating
+Apache Commons Text in these images to the latest version. Some of these images may not be
+vulnerable for other reasons. We recommend that you also review the guidelines published on the upstream websites.
+
+| Repository                | Patched version         | Additional documentation       |
+|:------------------------|:-----------------------|:-----------------------|
+| [bonita](https://hub.docker.com/_/bonita) | Awaiting info | Awaiting info |
+| [Couchbase](https://hub.docker.com/_/couchbase) | Awaiting info | Awaiting info |
+| [Geonetwork](https://hub.docker.com/_/geonetwork) | Awaiting info | Awaiting info |
+| [neo4j](https://hub.docker.com/_/neo4j) | Awaiting info | Awaiting info |
+| [sliverpeas](https://hub.docker.com/_/sliverpeas) | Awaiting info | Awaiting info |
+| [solr](https://hub.docker.com/_/solr) | Awaiting info | Awaiting info |
+| [xwiki](https://hub.docker.com/_/xwiki) |  Awaiting info | Awaiting info |
```

Comment:
Are we going to merge this before all this is filled in or are we going to wait?